### ISSUE TITLE
Split data into train/validate and test

### ISSUE THREAD

**Issue by avishkar58** [avishkar58/irdm-collab-filtering#3]
Currently just using the top 10000 lines as a single training set. Need to split all of the data into a train, validation and test set (in appropriate proportions)
